ow_plexil/src/plexil-adapter/OwInterface.h

```diff
@@ -209,10 +217,12 @@ class OwInterface
   std::unique_ptr<DigCircularActionClient> m_digCircularClient;
   std::unique_ptr<DigLinearActionClient> m_digLinearClient;
   std::unique_ptr<DeliverActionClient> m_deliverClient;
+  std::unique_ptr<IdentifySampleLocationActionClient> m_identifySampleLocationClient;
 
   // Antenna state - note that pan and tilt can be concurrent.
   double m_currentPan, m_currentTilt;
   double m_goalPan, m_goalTilt;      // commanded pan/tilt values
+  bool m_pointCloudRecieved;
```

Comment:
This should be initialized in the default constructor (even though in current usage it doesn't seem to matter.